doc/source/reference/distutils.rst

```diff
@@ -117,6 +120,8 @@ install the C library, you just use the method `add_installed_library` instead o
 `add_library`, which takes the same arguments except for an additional
 ``install_dir`` argument::
 
+  >>> import numpy.distutils.misc_util
+  >>> config = np.distutils.misc_util.Configuration(None, '', '.')
```

Comment:
I think we either need to adjust the text to explain these lines, or hide them in a comment like the one at the top

Reply:
I hesitated between the two. I can move both lines to the hidden block at the top.